core/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/core/schedule/JobSchedulerMetrics.kt

```diff
@@ -48,7 +48,7 @@ class JobSchedulerMetrics : ToXContentFragment, Writeable {
 
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
         if (lastExecutionTime != null)
-            builder.timeField("last_execution_time", "last_execution_time", Instant.ofEpochMilli(lastExecutionTime).toEpochMilli())
+            builder.timeField("last_execution_time_millis", "last_execution_time", Instant.ofEpochMilli(lastExecutionTime).toEpochMilli())
```

Comment:
I thought we wanted to add `_in_millis` to the human readable parameter?

Reply:
yes, fixed